src/sentry/stacktraces/processing.py

```diff
@@ -32,6 +32,9 @@ def __eq__(self, other: object) -> bool:
     def __ne__(self, other: object) -> bool:
         return self is not other
 
+    def get_frames(self) -> Sequence[dict[str, Any]]:
+        return get_path(self.stacktrace, "frames", filter=True, default=())
```

Comment:
`get_path` is untyped -- we should be explicit about this instead of using `get_path`